src/server.js

```diff
@@ -0,0 +1,35 @@
+import 'dotenv/config'
+import { WebSocketServer } from 'ws'
+import { v4 as uuid } from 'uuid'
+import ZmqConnection from './zmq.js'
+import Client from './client.js'
+import Dashcore from '@dashevo/dashcore-lib'
+
+export default class Server {
+  #clients = []
+
+  constructor () {
+    this.zmq = new ZmqConnection()
+    this.zmq.on('rawtx', (data) => {
+      this.handleRawTransactions(data)
+    })
+  }
+
+  async start () {
+    const socket = new WebSocketServer({ port: process.env.PORT })
+
+    socket.on('connection', (socket) => {
+      socket.id = uuid()
```

Comment:
I feel like this uuid more related to Client, rather than socket, since it is user-generated. It is also a best practice to not mutate arguments and objects you do not own (that field could be used internally by the library)